Phoenix/VarnishCache/sql/phoenix_varnishcache_setup/upgrade-3.0.0-3.0.1.php

```diff
@@ -0,0 +1,17 @@
+<?php
+$installer = $this;
+$installer->startSetup();
+$installer->setConfigData('system/varnishcache/disable_routes', 'checkout
+customer
```

Comment:
@willtran please change this so that the setup script reads the current set of routes and simply adds the new (e.g. ebizmarts) route to the end of whatever is already present in the specific database where the script is being executed. There are some projects using this module which have project-specific routes and we don't want to lose those routes.